### PR TITLE
IsAdmin defined in User.{cc,h}

### PR DESCRIPTION
**Pull request by czipperz**

### CODE REVIEW

**Comment by czipperz** [comment]
#38

**Comment by czipperz** [comment]
Not sure what I should be doing with my copyright notice.

**Comment by czipperz** [comment]
https://msdn.microsoft.com/en-us/library/windows/desktop/aa376389(v=vs.85).aspx

This requires Advapi32.lib to be linked with on Windows.  I don't know how to do that.

**Comment by dkrutsko** [comment]
Merged! My apologies once again for taking a year to get to this. Some notes: The code will be part of the new `System` class (see #50), along with other useful functionality (like #77). I'll take care of including the necessary dependencies.

For the copyright header, it's consistent across the entire project, I don't include names in the core modules (not even mine). I'm trying to figure out how to best acknowledge all the wonderful contributors. I know some projects just point to the GitHub [Contributors](https://github.com/Robot/robot/graphs/contributors) page, others point to a special thank you file. But since Robot consists of multiple closely related projects I was thinking of creating a special page on the robot [website](http://getrobot.net/) with links to everyone who contributed to the project. Please let me know if you have any feedback! And thanks once again :-D

**Comment by czipperz** [comment]
Ok cool.  Feel free to remove the copyright notice.  FYI The copyright on that file is not correct.  I'll comment on the commit

File: Source/User.cc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+////////////////////////////////////////////////////////////////////////////////
+// -------------------------------------------------------------------------- //
+//                                                                            //
+//                       (C) 2010-2016 Robot Developers                       //
+//                       (C) 2016 Chris Gregory czipperz@gmail.com            //
```
Comment (czipperz) [inline Source/User.cc]: This line should be deleted and the one above should be (C) 2016-2017.

**Comment by dkrutsko** [comment]
No problem! I'll correct the file as part of the integration process. It'll be part of #78.

**Pull request merged**